int16 steering
int16 throttle
int16 camera_servo
int16 drone_servo
